### PR TITLE
No more permissions craziness!

### PR DESCRIPTION
**Pull request by weaverryan**

### CODE REVIEW

File: setup/file_permissions.rst [hunk 1]
```diff
@@ -1,86 +1,18 @@
...
-
-.. note::
-
-    Changing the umask is not thread-safe, so the ACL methods are recommended
-    when they are available.
-
-.. _`enable ACL support`: https://help.ubuntu.com/community/FilePermissionsACLs
+    If you decide to store log files on disk, you *will* need to make sure your
+    logs directory (e.g. ``var/logs``) is writable by your web server user and
+    terminal user.
```
Comment (Nyholm) [inline setup/file_permissions.rst]: May be add how would I do that?

**Comment by weaverryan** [comment]
Added a note about 777'ing the logs if you need to. There are other ways to do this... but I don't think we can really go into a rabbit hole here with that :)

**Review by javiereguiluz — APPROVED** [review]

**Comment by weaverryan** [comment]
Thanks to the people who actually made this possible. I just got to do the fun part (delete the page)

**Comment by ostrolucky** [comment]
I am interested for reasoning why was this done. Can you post some references or/and explanation?

**Pull request merged**